### DISCUSSION TITLE
Running LibreHardwareMonitor as a service

### DISCUSSION THREAD

**Discussion by shalak** [LibreHardwareMonitor/LibreHardwareMonitor#1115] [Q&A]
Hello!

Is there a way to run LibreHardwareMonitor as a service on Windows 11, without user being logged in?

**Comment by jason-grimme** [comment]
You would need to provide more information on your end goal.  What do you want to happen when it runs?
Simply running the tool as a service won't accomplish much, 

To run at startup, you could have a scheduled task.  
To run as system account (`nt authority\system`) you could use [psexec ](https://learn.microsoft.com/en-us/sysinternals/downloads/psexec) to run the app as system with interactive mode 
```
psexec.exe -i -s C:\path\to\LibreHardwareMonitor.exe
```

**Reply by shalak** [reply]
The goal is to have a persistent LHM process, that will be available via web interface, even before a user logs in.

**Reply by Spamfast** [reply]
I'd like this as well now there's a Home Assistant integration so I can monitor my Windows/NextPVR media PC which is on all the time with nobody logged in.

**Reply by ace-comp** [reply]
You could use the older version 0.9.1 of LHM which runs independently from a user login. I don't know why this has been removed in a later version as it's very useful for some scenarios.

**Comment by darkbrain-fc** [comment]
You can try my project: https://github.com/darkbrain-fc/HardwareSupervisor

**Reply by Spamfast** [reply]
Thanks. I'll give it a go.
